src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -24,12 +24,12 @@
 [assembly: InternalsVisibleTo("OrleansRuntime")]
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
+[assembly: InternalsVisibleTo("OrleansAWSUtils")]
```

Comment:
Why is this needed? In newer providers (such as the event hub stream provider) we started doing some work so that it doesn't require any internals, and any external provider could also be developed in the same way as built in ones. Is it possible to avoid this, and if not, let's try to find a way to make it better. 
The event hub provider did require some refactoring of the main Orleans assemblies, but in the end it improved them for all stream providers.